samples/remote_attestation/README.md

```diff
@@ -43,7 +43,7 @@ There are two types of attestation:
 
 ### Secure Communication Channel
 
```

Comment:
It would be great to add explanation to cover the mechanism of secure establishment.